src/pages/EditResourcePage.tsx

```diff
@@ -18,29 +22,71 @@ const StyledEditPublication = styled.div`
 `;
 
 interface EditResourcePageParamTypes {
-  identifier: string;
+  resourceIdentifierFromParam: string;
 }
 
 const EditResourcePage: FC = () => {
-  const { identifier } = useParams<EditResourcePageParamTypes>();
-  const [expanded, setExpanded] = useState<string | false>(false);
   const { t } = useTranslation();
+  const { resourceIdentifierFromParam } = useParams<EditResourcePageParamTypes>();
+
+  const [resourceIdentifier, setResourceIdentifier] = useState<string>(resourceIdentifierFromParam);
+  const [expanded, setExpanded] = useState<string | false>(false);
```

Comment:
er det ikke dette enklere? tom streng er jo false (kan evt bruke `!!expanded` også)
```suggestion
  const [expanded, setExpanded] = useState('');
```

Reply:
https://github.com/BIBSYSDEV/NVA-Frontend/blob/9e3a02ef02e2c9022d2f7e4b1d4112e2e7939af9/src/pages/registration/EditRegistration.tsx#L17

Reply:
går for ren streng, ja